internal/gen/controller.swagger.json

```diff
@@ -1,226 +1,171 @@
 {
-  "swagger": "2.0",
```

Comment:
Unfortunately there does not seem to be a way to avoid the contents of this file being completely rearranged. It isn't important, but it is a bit annoying.